### ISSUE TITLE
Updating order status to shipped changes the shipping & tax adjustment to "included"

### ISSUE THREAD

**Issue by matthisamoto** [craftcms/commerce#1272]
### Description

Having the same problems as #1243 however was asked to open a new issue.

Order Details are changing the price on change of order status, showing as Overpaid because Shipping and Tax adjustments are no longer being applied correctly, showing as included when the rule set does not have them included.

### Steps to reproduce

1. Set order status to Shipped from Processing.
2. Order Details appear to count the shipping and tax charges as included only once this status change happens. We do not have the tax rule set to be included, it is separated. 
3. The Order Details page also appears to be incorrectly displaying the Shipping adjustment total as the per item rate ($2) as well, the Ground shipping total minimum cost is $4, which coupled with the $1.65 Tax adjustment accounts for the discrepancy in displayed Total Price at $20 vs Paid Amount at $25.65.

![](https://user-images.githubusercontent.com/547172/74046333-f81ea700-4993-11ea-8f21-5ef893af7cb4.png)

![Ground Shipping Method Settings](https://user-images.githubusercontent.com/547172/74047210-97906980-4995-11ea-9a17-b88646a4b43d.png)

We recently did a Craft 2 => 3 update within the last month, which took us to Commerce 2, then did the Commerce 3 update the other week when it was released.

**Application Info**
PHP version	7.3.14
OS version	Linux 3.10.0-957.27.2.el7.x86_64
Database driver & version	MySQL 5.7.29
Image driver & version	GD 7.3.14
Craft edition & version	Craft Pro 3.4.3
Yii version	2.0.32
Twig version	2.12.3
Guzzle version	6.5.1
Imagine version	1.2.3-dev

**Plugins**
Calendar	3.0.0
Cookies	1.1.11
Craft Commerce	3.0.5
Feed Me	4.2.0.1
Field Manager	2.1.1
Imager	v2.3.1
Many to Many Field Type	1.0.2.2
Navigation	1.3.2
Redactor	2.5.0
Retour	3.1.33
Smart Map	3.2.2.1
Stripe for Craft Commerce	2.2.3
SuperSort	3.0.1
Typogrify	1.1.18

**Comment by lukeholder** [comment]
This issue looks like it was fixed? Can you test again with the `included` bug now fixed?

**Comment by matthisamoto** [comment]
I was just doing as instructed by your comment on the issue, I'm confused as to what you need: https://github.com/craftcms/commerce/issues/1243#issuecomment-583562987

**Comment by lukeholder** [comment]
Change your `craftcms/commerce` requirement in composer.json to:

```json
"require": {
  "craftcms/commerce": "dev-develop#b9974fbd92e5feaa129e8166b6ede74bc174913b as 3.0.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by lukeholder** [comment]
After you have tested above, let me know if the calculations look wrong still.

**Comment by matthisamoto** [comment]
Updated via composer, it ran the DB "Finishing up" afterward, however the order is identical to the screenshot I shared, and it seems it is still incorrect.

![Price-Paid Discrepancy](https://user-images.githubusercontent.com/547172/74061047-cff17100-49b0-11ea-9166-aac4b1197afe.png)

![Order Adjustments](https://user-images.githubusercontent.com/547172/74061044-cf58da80-49b0-11ea-9982-abdc0ff3e38b.png)

```
  - Removing craftcms/commerce (3.0.5)
  - Installing craftcms/commerce (dev-develop b9974fb): Cloning b9974fbd92 from cache
```

**Comment by lukeholder** [comment]
Is that a new order or the same order that was affected by the original bug?

The fix stops the issue from occurring for new order being edited.

**Comment by matthisamoto** [comment]
Okay. It is the same order. Is there some way I can edit these few to get them to appear correct or should I just tell the client to ignore it? As far as I can tell it affects nothing regarding the payment or anything.

**Comment by lukeholder** [comment]
You can click edit on affected orders and change untick the included checkboxes for adjustments and save the order. This will no break anything but fix totals etc.

**Comment by matthisamoto** [comment]
Thanks @lukeholder. Will the new update show up in Updates when it's released or will I have to manually change my composer.json back to pick it up?

**Comment by lukeholder** [comment]
@matthisamoto you will need to change your composer.json back when the release is out.
